src/containers/RuleContainer.jsx

```diff
@@ -140,94 +61,38 @@ const RuleContainer = ({ index, editParameters }) => {
-        dispatch(
-            MappingSlice.actions.addFilter({
-                ruleIndex: index,
-                groupIndex,
-                groupOperator,
-            })
-        );
-
-    const editGroup = (isAbsolute) => () =>
-        editParameters({
-            model: mappedModel,
-            setGroup,
-            groupType,
-            isAbsolute,
-            origin: GroupEditionOrigin.RULE,
-            originIndex: index,
-        });
-
-    function buildFilters() {
-        const filters = [];
-        for (let i = 0; i < filtersNumber; i++) {
-            filters.push(
-                <FilterContainer
-                    key={`filter-container-${i}`}
-                    ruleIndex={index}
-                    filterIndex={i}
-                    equipmentType={type}
-                />
-            );
-        }
-        return filters;
-    }
-
-    function buildFiltersGroup(groupArray, groupIndex) {
-        const groupOperator = groupArray[1] ?? '||';
-        const filters = groupArray
-            .filter((elt, index) => index % 2 === 0 && elt !== 'true')
-            .map((filterId) => (
-                <FilterContainer
-                    key={`filter-container-${filtersIndexMap[filterId]}`}
-                    ruleIndex={index}
-                    filterIndex={filtersIndexMap[filterId]}
-                    equipmentType={type}
-                />
-            ));
-        const changeGroupOperator = (newValue) => {
-            let operatorIndexes = [];
-            for (let i = 0; i < groupArray.length / 2 - 1; i++) {
-                operatorIndexes.push(2 * i + 1);
-            }
-            changeCompositionFromArray(groupIndex)(newValue, operatorIndexes);
-        };
+    const deleteRule = useCallback(
+        () =>
+            dispatch(
+                MappingSlice.actions.deleteRule({
+                    index,
+                })
+            ),
+        [dispatch, index]
+    );
 
-        return (
-            <FiltersGroup
-                filters={filters}
-                groupOperator={groupOperator}
-                changeGroupOperator={changeGroupOperator}
-                addFilter={addFilterInGroup(groupIndex, groupOperator)}
-                key={`group-${groupIndex}`}
-            />
-        );
-    }
+    const copyRule = useCallback(
+        () =>
+            dispatch(
+                MappingSlice.actions.copyRule({
+                    index,
+                })
+            ),
+        [dispatch, index]
+    );
 
-    // TODO intl
-    const noFilterLabel = 'no other rule applies';
+    const editGroup = useCallback(
+        (isAbsolute) => () =>
+            editParameters({
+                model: mappedModel,
+                setGroup,
+                groupType,
+                isAbsolute,
+                origin: GroupEditionOrigin.RULE,
+                originIndex: index,
+            }),
+        [editParameters, index]
```

Comment:
[eslint]
 React Hook useCallback has missing dependencies: 'groupType', 'mappedModel', and 'setGroup'. Either include them or remove the dependency array

Reply:
DONE in https://github.com/gridsuite/griddyna-app/pull/95/commits/96f55374b90f2220f3662f8ecd54627790745aa2